modules/nw-banp-audit-logging-concept.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_network_security/AdminNetworkPolicy/logging-anp-policy.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-banp-audit-logging-concept_{context}"]
+= BaselineAdminNetworkPolicy audit logging
+
+Audit logging is enabled in the `BaselineAdminNetworkPolicy` CR by annotating an ANP policy with the `k8s.ovn.org/acl-logging` key such as in the following example:
+
+.Example of annotation for `BaselineAdminNetworkPolicy` CR
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: BaselineAdminNetworkPolicy
+metadata:
+  annotations:
+    k8s.ovn.org/acl-logging: '{ "deny": "alert", "allow": "alert"}'
+  name: default
+spec:
+  subject:
+    namespaces:
+      matchLabels:
+          tenant: red
+  ingress:
+  - name: "default-allow-ns"
+    action: "Allow"
+    from:
+    - namespaces:
+          matchLabels:
+            tenant: blue
+  - name: "default-deny-ns"
+    action: "Deny"
+    from:
+    - namespaces:
+          matchLabels:
+            tenant: orange
+----
+====
+
+In the example, an event in which the `tenant: blue` is allowed ingress to the `tenant: red` namespace a log is generated.
```

Comment:
could you give a small preface like "tenant:blue" is set of all namespaces with the label "tenant:blue" ? same for red
so its about defining what we meant by tenant here as a set of colored namespaces, so blue tenant is a set of namespaces with the label tenant:blue and red tenant is the set of namespaces with the label "red"